frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -31,9 +33,24 @@ const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
   const {
     metadata: { name, namespace },
   } = pipeline;
+  const [showAlert, setShowAlert] = React.useState(isPipelineNotStarted(name));
+
+  React.useEffect(() => {
+    setShowAlert(isPipelineNotStarted(name));
+  }, [name]);
+
   return (
     <>
       <SidebarSectionHeading text={PipelineRunModel.labelPlural}>
+        {showAlert && (
```

Comment:
```suggestion
        {(showAlert && pipelineRuns.length === 0) && (
```
If they start up a Pipeline Run we don't want to show it.